code/modules/projectiles/ammunition/ballistic/lmg.dm

```diff
@@ -1,23 +1,23 @@
-// 1.95x129mm (SAW)
+// 5.56x28mm (SAW)
 
-/obj/item/ammo_casing/mm195x129
-	name = "1.95x129mm bullet casing"
-	desc = "A 1.95x129mm bullet casing."
+/obj/item/ammo_casing/mm556x45

```

Comment:
1.95 man good, 5.56 man bad.
Why ARE these calibres getting changed anyways?

Reply:
Why not something cool like 7.62x39mm (RPK ammunition) or 12.7x99mm (.50 BMG)? Those are cooler than boring NATO 5.56, IMO